### PR TITLE
Add signatures in OpenQA::Task::Bug::Limit and more

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by Martchus** [comment]
But considering the CI results there must be a syntax error.

File: lib/OpenQA/Utils.pm [hunk 1]
```diff
@@ -204,7 +201,7 @@ sub testcasedir {
 
 =head2 gitrepodir
 
-  gitrepodir(distri => DISTRI, version => VERSION)
+  gitrepodir({distri => DISTRI, version => VERSION})
```
Comment (perlpunk) [inline lib/OpenQA/Utils.pm:204]: Why?

Comment (okurz) [reply lib/OpenQA/Utils.pm:204]: I guess we will never know …

**Review by perlpunk — APPROVED** [review]
I assume you simply fixed the things we commented on and nothing else, I don't want to go through the whole diff of 10 commits again :(

**Pull request merged**